storage/AggregationOperationState.cpp

```diff
@@ -79,58 +78,73 @@ AggregationOperationState::AggregationOperationState(
     group_by_types.emplace_back(&group_by_element->getType());
   }
 
-  // Set up each individual aggregate in this operation.
```

Comment:
Note: there is no big change starting from here though the file diff shows large chunks of modification. The modifications are just
```
if (aggregate_functions.size() == 0) {
  // New code here
} else {
  // The original big chunk of code indented two more spaces.
}